tests/components/ring/test_switch.py

```diff
@@ -0,0 +1,116 @@
+"""The tests for the Ring sensor platform."""
+import os
+import unittest
+import requests_mock
+import asynctest
+from datetime import datetime
+
+
+import homeassistant.components.ring.switch as ring
+from homeassistant.components import ring as base_ring
+
+from tests.components.ring.test_init import VALID_CONFIG
+from tests.common import (
+    get_test_config_dir, get_test_home_assistant, load_fixture)
+
+
+class TestRingSensorSetup(unittest.TestCase):
```

Comment:
New tests should use standalone pytest test functions.

Reply:
Ok, for now I've removed the tests. as it's taking a bit to understand the new way (but I'm getting there). I'll add them as another commit later.

Reply:
These have now been updated to use the new format - any feedback would be ace!